lib/travis/build/script/elm.rb

```diff
@@ -0,0 +1,183 @@
+module Travis
+  module Build
+    class Script
+      class Elm < NodeJs
+        # Default NodeJS version to install
+        DEFAULT_NODE_VERSION = '8.9.1'
```

Comment:
Is this the preferred version? v10 has entered the LTS stage, so maybe that's preferable?

Reply:
Updated in 74f7059eab2e893a5cea513c06846070403feebe